<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab10_TNR</class>
 <widget class="QMainWindow" name="Lab10_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab10_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="minBtn_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Find min item</string>
    </property>
   </widget>
   <widget class="QLabel" name="minLabel_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min String</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToEndBtn_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to the tnd of list</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="item_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="posVal_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setAtBtn_tnr">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set at position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortABtn_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Ascending</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortDBtn_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Descending</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fillBtn_tnr">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill with random lines</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numLinesVal_tnr">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Num of lines:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Item:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randBtn_tnr">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="array_tnr">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>256</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn_tnr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>String_TNR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polynom_TNR</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
